kedro-telemetry/kedro_telemetry/plugin.py

```diff
@@ -145,6 +149,19 @@ def after_catalog_created(self, catalog):
         )
 
 
+def _check_is_known_ci_env():
+    # Most CI tools will set the CI environment variable to true
+    if os.getenv("CI") == "true":
```

Comment:
Are we confident it will be `"true"` and not `"True"` or `1`?

Reply:
I checked, it's "true" for most places.